packages/fe-fpm-writer/templates/section/manifest.section.json

```diff
@@ -0,0 +1,33 @@
+{
+    "sap.ui5": {
+        "routing": {
+            "targets": {
+                "<%- target %>": {
+                    "options": {
+                        "settings": {
+                            "content": {
+                                "body": {
+                                    "sections": {
+                                        "<%- name %>": {
+                                            "name": "<%- ns %>.<%- name %>",
+                                            "position": {
+                                                <% if (position.placement) { %> 
+                                                    "placement": "<%- position.placement %>", 
+                                                <% } %>
+                                                <% if (position.anchor) { %> 
+                                                    "anchor": "<%- position.anchor %>" 
+                                                <% } %>
+                                            },
+                                            "type": "<%- type %>",
```

Comment:
This can also be removed since Fiori elements is not evaluating the property, correct?

Reply:
yes, deleted in commit 1a33177